tests/python/unittest/test_crt.py

```diff
@@ -106,6 +105,20 @@ def test_compile_runtime():
         assert (C_data.asnumpy() == np.array([6, 7])).all()
 
 
+@tvm.testing.requires_micro
+def test_compile_runtime_llvm():
+    """Test targeting the on-device runtime with the llvm backend."""
+    global TARGET
+    old_target = TARGET
+    try:
+        TARGET = tvm.target.Target("llvm " + str(TARGET)[2:])
```

Comment:
Not very clear to me what is going on here ?
I would imagine we would want to test both llvm and c targets, right ?
May I ask why is the need for try-finally here?

Reply:
clarified a bit--i'm being lazy and reusing test_compile_runtime